frontends/cypress_e2e_tests.sh

```diff
@@ -2,27 +2,25 @@
 
 set -x
 
+CYPRESS_TEST_IMAGE="quay.io/cloudservices/cypress-e2e-image:06b70f3"
+
+source ./helpers/general.sh
+
+trap_proxy teardown EXIT ERR SIGINT SIGTERM
+
 docker login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
 
 docker run \
     --name $TEST_CONT \
+    -v $PWD:/e2e:ro,Z \
     -e CHROME_ACCOUNT=$CHROME_ACCOUNT \
     -e CHROME_PASSWORD=$CHROME_PASSWORD \
     --add-host stage.foo.redhat.com:127.0.0.1 \
     --add-host prod.foo.redhat.com:127.0.0.1 \
-    -d -i --rm  \
+    --rm \
     --entrypoint bash \
-    "${CYPRESS_TEST_IMAGE}"
-
-docker cp -a . "${TEST_CONT}:/e2e/"
-
-docker exec -i $TEST_CONT sh -c "npm i"
-docker exec -i $TEST_CONT sh -c "npm run test:e2e"
+    "${CYPRESS_TEST_IMAGE}" /e2e/run-e2e.sh
 
 RESULT=$?
 
-if [[ $RESULT -ne 0 ]]; then
-    exit $RESULT
-fi
-
-docker stop $TEST_CONT
+exit $RESULT
```

Comment:
you can safely delete all lines from line 23, the script will exit on the latest command's exit code